static/i18n/en/app.json

```diff
@@ -19,7 +19,7 @@
     },
     "app": {
       "informations": {
```

Comment:
The key should be renamed as well, is it possible or would it wreak havoc with Crowdin translating platform?

Reply:
Good point. Ideally it would be consistent but probably not worth spending hours on if it does break things. Will you get errors before trying to run the app or would they only be caught by using the app?

Reply:
If the key is updated alongside all of the places it is used in the app (today only one place) everything will work fine. But since these i18n files are generated and managed by a third party service (Crowdin), I don't know how these kind of key changes are managed (if they are)